controlplane/src/db/schema.ts

```diff
@@ -1012,3 +1012,58 @@ export const subgraphMembers = pgTable(
     };
   },
 );
+
+export const discussions = pgTable('discussions', {
+  id: uuid('id').notNull().primaryKey().defaultRandom(),
+  createdAt: timestamp('created_at', { withTimezone: true }).notNull().defaultNow(),
+  targetId: uuid('target_id')
+    .references(() => targets.id, {
+      onDelete: 'cascade',
+    })
+    .notNull(),
+  schemaVersionId: uuid('schema_version_id')
+    .notNull()
+    .references(() => schemaVersion.id, {
+      onDelete: 'cascade',
+    }),
+  referenceLine: integer('reference_line').notNull(),
+  isResolved: boolean('is_resolved').default(false).notNull(),
+});
+
+export const discussionThread = pgTable('discussion_thread', {
+  id: uuid('id').notNull().primaryKey().defaultRandom(),
+  discussionId: uuid('discussion_id')
+    .notNull()
+    .references(() => discussions.id, {
+      onDelete: 'cascade',
+    }),
+  contentMarkdown: text('content_markdown'),
+  contentJson: customJson('content_json'),
```

Comment:
You can specify a type here to be type safe during development.

Reply:
Done